src/main/resources/swagger/agora.json

```diff
@@ -1,1353 +1,1442 @@
+                        "description": "Type of Method - Task or Workflow",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "excludedFields",
+                        "in": "query",
+                        "description": "Fields to exclude in JSON response.",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "includedFields",
+                        "in": "query",
+                        "description": "Fields to include in JSON response",
+                        "required": false,
+                        "type": "string"
+                    }
+                ],
+                "tags": [
+                    "Methods"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "An array of methods.",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/MethodResponse"
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error",
+                        "schema": {
+                            "$ref": "#/definitions/Error"
+                        }
+                    }
+                }
+            },
+            "post": {
+                "summary": "Add a Method.",
+                "description": "Inserts a method into the method repository and\ngives the owner full access permissions. Must supply a namespace\nname, payload, and entityType. Cannot supply a snapshotId\n",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "description": "Agora Entity",
+                        "required": false,
+                        "schema": {
+                            "$ref": "#/definitions/MethodQuery"
+                        }
+                    }
+                ],
+                "tags": [
+                    "Methods"
+                ],
+                "responses": {
+                    "201": {
+                        "description": "Method successfully added.",
+                        "schema": {
+                            "$ref": "#/definitions/MethodResponse"
+                        }
+                    },
+                    "400": {
+                        "description": "Malformed input.",
+                        "schema": {
+                            "$ref": "#/definitions/Error"
```

Comment:
The $ref here is not correctly associated with this response. Error is a 500 message, whereas this is 400. We need another definition for malformed response. Probably ones for 401 unauthorized and 404 not found as well